spec/factories/form526_job_statuses.rb

```diff
@@ -11,15 +11,22 @@
     bgjob_errors { {} }
   end
 
+  # rubocop:disable Style/StringLiterals
+  # rubocop:disable Layout/HashAlignment
+  # rubocop:disable Layout/SpaceAroundOperators
+  # rubocop:disable Style/RedundantInterpolation
```

Comment:
These all seem like valid offenses. Why are these being ignored?

Reply:
> These all seem like valid offenses. Why are these being ignored?

I think locally disabling rubocop rules here is justified since these are (slightly) modified hashes taken form the logs and I don't see much point in reformatting them.